firebase-app-distribution/api.txt

```diff
@@ -1,14 +1,150 @@
+  public class FirebaseAppDistributionException extends com.google.firebase.FirebaseException {
+    ctor public FirebaseAppDistributionException(@NonNull String, @NonNull com.google.firebase.app.distribution.FirebaseAppDistributionException.Status);
+    ctor public FirebaseAppDistributionException(@NonNull String, @NonNull com.google.firebase.app.distribution.FirebaseAppDistributionException.Status, @Nullable com.google.firebase.app.distribution.AppDistributionRelease);
+    ctor public FirebaseAppDistributionException(@NonNull String, @NonNull com.google.firebase.app.distribution.FirebaseAppDistributionException.Status, @NonNull Throwable);
+    ctor public FirebaseAppDistributionException(@NonNull String, @NonNull com.google.firebase.app.distribution.FirebaseAppDistributionException.Status, @Nullable com.google.firebase.app.distribution.AppDistributionRelease, @NonNull Throwable);
+    method @NonNull public com.google.firebase.app.distribution.FirebaseAppDistributionException.Status getErrorCode();
+    method @Nullable public com.google.firebase.app.distribution.AppDistributionRelease getRelease();
+  }
+
+  public enum FirebaseAppDistributionException.Status {
+    enum_constant public static final com.google.firebase.app.distribution.FirebaseAppDistributionException.Status APP_RUNNING_IN_PRODUCTION;
+    enum_constant public static final com.google.firebase.app.distribution.FirebaseAppDistributionException.Status AUTHENTICATION_CANCELED;
+    enum_constant public static final com.google.firebase.app.distribution.FirebaseAppDistributionException.Status AUTHENTICATION_FAILURE;
+    enum_constant public static final com.google.firebase.app.distribution.FirebaseAppDistributionException.Status DOWNLOAD_FAILURE;
+    enum_constant public static final com.google.firebase.app.distribution.FirebaseAppDistributionException.Status INSTALLATION_CANCELED;
+    enum_constant public static final com.google.firebase.app.distribution.FirebaseAppDistributionException.Status INSTALLATION_FAILURE;
+    enum_constant public static final com.google.firebase.app.distribution.FirebaseAppDistributionException.Status INSTALLATION_FAILURE_SIGNATURE_MISMATCH;
+    enum_constant public static final com.google.firebase.app.distribution.FirebaseAppDistributionException.Status NETWORK_FAILURE;
+    enum_constant public static final com.google.firebase.app.distribution.FirebaseAppDistributionException.Status RELEASE_URL_EXPIRED;
+    enum_constant public static final com.google.firebase.app.distribution.FirebaseAppDistributionException.Status UNKNOWN;
+    enum_constant public static final com.google.firebase.app.distribution.FirebaseAppDistributionException.Status UPDATE_NOT_AVAILABLE;
+  }
+
+  public class FirebaseAppDistributionFileProvider extends androidx.core.content.FileProvider {
+    ctor public FirebaseAppDistributionFileProvider();
+  }
+
+  public class FirebaseAppDistributionLifecycleNotifier implements android.app.Application.ActivityLifecycleCallbacks {
```

Comment:
@mannyjimenez0810 out of curiosity, could \ we make this Lifecycle class package private? and perhaps the interfaces as well?